changelogs/fragments/261-scrub-params-descend-into-lists.yml

```diff
@@ -0,0 +1,4 @@
+---
+minor_changes:
+  - module_utils/core - add parameter ``descend_into_lists`` to ``scrub_none_parameters`` helper function. Defaults to
+    True as the function has yet to be implemented anywhere.
```

Comment:
Since also module_utils are part of the public API of a collection, this is a breaking change as it could have been used in a third-party collection/module/plugin. Better set the default value to `False` to avoid the hassle of having to wait for a new major release.

Reply:
We'll have a major release in the next few months, so we could plan to change the default at that time.

Reply:
Once we get this merged I'll prep a 'breaking' change which we can merge once the decision is made that the next release is 2.0.0